modules/gitops-release-notes-1-10-1.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-10-1_{context}"]
+= Release notes for {gitops-title} 1.10.1
+
+{gitops-title} 1.10.1 is now available on {OCP} 4.12 and 4.13.
+
+[id="errata-updates-1-10.1_{context}"]
+== Errata updates
+
+[id="gitops-1-10-1-security-update-advisory_{context}"]
+=== RHSA-2023:XXXX - {gitops-title} 1.10.1 security update advisory
+
+Issued: 2023-10-30
```

Comment:
@masauso-lungu  Please change the release date to Oct 31. 

1.10.1 is getting released tomorrow.

Reply:
I have updated the release date. Thank you.